utils/ffmpeg_install.py

```diff
@@ -60,7 +58,8 @@ def ffmpeg_install():
     try:
         # Try to run the FFmpeg command
         subprocess.run(['ffmpeg', '-version'], check=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        print('FFmpeg is installed on this system! If you are seeing this error for the second time, restart your computer.')
+        if not os.path.exists("./results") :
```

Comment:
Why this if statement?

Reply:
i will take a look

Reply:
to check if person is newly installing the bot